src/components/navigation-bar/navigation-bar.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import NavigationBar from './navigation-bar';
+
+configure({ adapter: new Adapter() });
+
+describe('navigation-bar Component', () => {
+  const navigationBar = mount(<NavigationBar />);
+
+  it('should not be null', () => {
+    expect(navigationBar).not.toBeNull();
+    expect(navigationBar.find('.navigation-bar').length).toEqual(1);
+  });
+
+  it('ensure that events content is in the tab component', () => {
```

Comment:
consider adding `should` to both these `it` blocks, so that it reads more naturally, e.g.
> it _should_ ensure that etc...